packages/models-library/src/models_library/services.py

```diff
@@ -359,7 +402,7 @@ class ServiceAccessRights(BaseModel):
 
 class ServiceMetaData(ServiceCommonData):
     # Overrides all fields of ServiceCommonData:
-    #    - for a partial update all members must be Optional
+    #    - for a partial update all members must be Optional FIXME: create a different model for updates!
```

Comment:
this is indeed a bit confusing

Reply:
if API entry needs a body to allow partial updates (e.g. patch/put),  it should be implemented with a different model e.g. ServiceMetaDataUpdate